platform/native/contiki-main.c

```diff
@@ -242,6 +242,8 @@ main(int argc, char **argv)
 
   serial_line_init();
 
+  process_start(&sensors_process, NULL);
```

Comment:
I would not start this for any native target. Instead the projects can include the sensors based on their preferences.